src/client/components/RobotLogo/RobotLogo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Robot from '../../assets/images/robot_logo.png';
+import Arrow from '../../assets/images/arrow.png';
+import './RobotLogo.css';
+
+export default function RobotLogo() {
+  return (
+    <div className="RobotLogo">
+      <div className="RobotLogoDiv">
+        <img src={Robot} alt="RobotLogo" />
+        <p>
+          CARRER FOR <br />
+          YOU IS
+        </p>
+      </div>
+      <div className="arrow">
+        <img src={Arrow} alt="RobotArrow" />
+      </div>
+    </div>
+  );
+}
```

Comment:
Add proptypes in here, like this - SampleComponent.propTypes = {
  title: PropTypes.string.isRequired,
  onClick: PropTypes.func,
  children: PropTypes.node,
};

SampleComponent.defaultProps = {
  onClick: null,
  children: null,
};

Reply:
thanks Beth :) done

Reply:
Sorry Aijaz, the given Proptypes was just an example. You need to make it specific to your code. Remove title, and add in Arrow and Robot. You also need to change from onClick to handleOnClick. and remove children proptype, as you haven't used it.

Reply:
yea done this